site/en/query/language.md

```diff
@@ -902,6 +902,15 @@ linkshared attribute (such as, `cc_binary` rule) and have it either
 explicitly set to 0 or do not set it at all but default value is 0 (such as for
 `cc_binary` rules).
 
+The values `True` and `False` set for a boolean attribute are converted to 
```

Comment:
On my first read I wasn't sure if the pre-converted scenario was referring to the target definitions or the query expression. Can you be a bit more explicit on that?